src/test/java/teammates/storage/sql/CoursePostgresDbTest.java

```diff
@@ -0,0 +1,79 @@
+package teammates.storage.sql;
+
+import org.testng.annotations.Test;
+import teammates.common.datatransfer.sqlattributes.CourseAttributes;
+import teammates.common.exception.EntityAlreadyExistsException;
+import teammates.common.exception.InvalidParametersException;
+import teammates.common.util.FieldValidator;
+import teammates.common.util.StringHelperExtension;
+import teammates.test.AssertHelper;
+import teammates.test.BaseTestCaseWithLocalPostgresSqlDatabaseAccess;
+
+/**
+ * SUT: {@link CoursesDb}.
+ */
+public class CoursePostgresDbTest extends BaseTestCaseWithLocalPostgresSqlDatabaseAccess {
```

Comment:
The tests are already in the `sql` folder, so the class names can be kept the same as their counterparts in the `api` folder (`CoursesDbTest`) . This reduces the need for us to refactor the class names again down the line. Same applies to `EntitiesDbTest`.

Reply:
Fixed